packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -15,13 +15,14 @@ import UnderReviewLabel from './components/UnderReviewLabel'
 const StyledBox = styled(Box)`
   position: relative;
 `
+const environment = process.env.APP_ENV
 
 function ProjectHeader (props) {
   const { className, inBeta, navLinks, screenSize, title } = props
 
   // hard-coded for translation feature PR testing, but
   // should eventually be imported as props instead
-  const availableLocales = ['en', 'test']
+  const availableLocales = ['en', 'fr', 'es', 'test']
```

Comment:
This could get the array of language codes from `project.available_languages`.

Reply:
Yes, `availableLocales` is already pulled into ProjectHeaderContainer it's just not passed to ProjectHeader yet.

Reply:
Done!